lib/pf/services/manager/haproxy_admin.pm

```diff
@@ -154,8 +154,8 @@ frontend admin-https-$mgmt_cluster_ip
         http-request redirect location /admin/alt if { lua.redirect 1 }
 EOT
             if (isenabled($Config{services}{httpd_admin})) {
-	$tags{'http_admin'} .= <<"EOT";
-	default_backend  $mgmt_cluster_ip-admin
+        $tags{'http_admin'} .= <<"EOT";
+        default_backend  $mgmt_cluster_ip-admin
```

Comment:
Is that file suppose to be pushed with that PR ?

I see it's not a real difference in the code but still.